Add cherry pick script from PT

huydhn opened the pull request:
The copy from third_party/pytorch doesn't work because it points to PyTorch

guangy10 commented on .github/scripts/cherry_pick.py:
```diff
@@ -0,0 +1,223 @@
+
+from github_utils import gh_fetch_url, gh_post_pr_comment
+
+from gitutils import get_git_remote_name, get_git_repo_dir, GitRepo
```
This doesn't exist for ET

guangy10 commented:
@huydhn can you link a result of local/manual run? Just to ensure we are not spotting new issues after waiting for this one to be landed

huydhn committed:
Add cherry pick script from PT (#2716)

Summary:
The copy from third_party/pytorch doesn't work because it points to PyTorch


Reviewed By: guangy10

Differential Revision: D55434703

Pulled By: huydhn
